install/kubernetes/cilium/templates/validate.yaml

```diff
@@ -172,6 +177,17 @@
     {{ fail "External workloads support cannot be enabled in combination with .Values.disableEndpointCRD=true" }}
   {{- end }}
 {{- end }}
+{{- if and .Values.clustermesh.apiserver.kvstoremesh.enabled (eq .Values.clustermesh.apiserver.kvstoremesh.kvstoreMode "external") }}
```

Comment:
Nit. Worth checking that `.Values.clustermesh.useAPIServer` is enabled as well, to avoid triggering an error if the whole feature is disabled.